package.json

```diff
@@ -30,6 +30,10 @@
     "typescript": "^5.5.4",
     "vitest": "^2.0.4",
     "webpack": "^5.93.0",
-    "xo": "^0.59.2"
+    "xo": "^0.59.3"
+  },
+  "dependencies": {
+    "open": "^10.1.0",
```

Comment:
Are these needed?

Reply:
I'll remove it. Was required for some packages that is not working now.